examples/unstructured_2d_fdsbp/elixir_euler_free_stream_upwind.jl

```diff
@@ -0,0 +1,82 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the compressible Euler equations
+
+equations = CompressibleEulerEquations2D(1.4)
+
+initial_condition = initial_condition_constant
+
+# Boundary conditions for free-stream preservation test
+boundary_condition_free_stream = BoundaryConditionDirichlet(initial_condition)
+
+boundary_conditions = Dict(:outerCircle => boundary_condition_free_stream,
+                           :cone1 => boundary_condition_free_stream,
+                           :cone2 => boundary_condition_free_stream,
+                           :iceCream => boundary_condition_free_stream)
+
+###############################################################################
+# Get the Upwind FDSBP approximation space
+
+# Note, one must set `xmin=-1` and `xmax=1` due to the reuse
+# of interpolation routines from `calc_node_coordinates!` to create
+# the physical coordinates in the mappings.
```

Comment:
Okay, that sounds reasonable. I thought about throwing an error if a user attempts to set anything different for the `xmin` and `xmax` values. Would this maybe be overkill? Alternatively, we could maybe add "under the hood" an affine map of the user desired `xmin` and `xmax` to the [-1,1] interval. However, I am hot sure if this would mess with plotting or post processing later.